README.md

```
@@ -220,6 +234,93 @@ skillsdock add ./my-skills --scope project
+skillsdock check
+
+# machine-readable output
+skillsdock check --json
+
+# with authentication for higher rate limits
+GITHUB_TOKEN=ghp_xxx skillsdock check
+```
+
+### Text Output
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)